src/opt/vyatta/share/vyatta-op/templates/clear/interfaces/connection/node.tag/node.def

```diff
@@ -0,0 +1,53 @@
+help:  Bring datalink layer connection down and back up
+
+allowed: local -a array ;
+         array=( /etc/ppp/peers/pppoe* /etc/ppp/peers/pppoa* /etc/ppp/peers/wan* /etc/ppp/peers/wlm* ) ;
+         echo  -n ${array[@]##*/}
+
+run: (	
+	IFNAME=${4}
+	LOGFILE=/var/log/vyatta/ppp_${IFNAME}.log
+        if [ ! -e /etc/ppp/peers/$IFNAME ]; then
+            echo "Invalid interface: $IFNAME"
+	    builtin exit 1
+	fi
+
+	# Serial PPP interfaces are started with "pon", which
```

Comment:
We can remove this block since serial WAN is not supported in VyOS anyway.

Reply:
Added by accident - rewritten by proper XML logic. Thanks!